v1.0/pvp-specification.md

```diff
@@ -182,13 +182,10 @@ Dependencies in Cabal
 
 When publishing a Cabal package, you **SHALL** ensure that your
 dependencies in the `build-depends` field are accurate. This means
-specifying not only lower bounds, but also upper bounds on every
-dependency.
-
-At some point in the future, Hackage may refuse to accept packages that
-do not follow this convention. The aim is that before this happens, we
-will put in place tool support that makes it easier to follow the
-convention and less painful when dependencies are updated.
+at least specifying lower bounds. You **SHALL* at least specify upper
+bounds if there are known incompatibilities.
+It is **NOT REQUIRED** to set an upper bound if all currently known
+versions of a dependency are compatible, although it is **RECOMMENDED**.
```

Comment:
We could also talk about the caret operator here, but I think that is for another PR and a different discussion.